test_regress/t/t_vlt_warn_bad.out

```diff
@@ -0,0 +1,5 @@
+%Warning-DEPRECATE: t/t_vlt_warn_bad.vlt:11: Deprecated -msg in configuration files, use -rule instead.
+lint_off -msg WIDTH          -file "*/t_vlt_warn.v" -lines 19-19
+^~~~~~~~
```

Comment:
Note the arrow points at the wrong thing. This means you used the wrong token (not the --msg) for the warning.